src/coffee/connect/rest.coffee

```diff
@@ -193,7 +193,7 @@ class Rest
     throw new Error 'Query limit doesn\'t seem to be 0. This function queries all results, are you sure you want to use this?' if query.limit and query.limit isnt '0'
 
     params = _.extend {}, query,
-      limit: 50 # limit used for batches
+      limit: 500 # limit used for batches
```

Comment:
Looking at your comment in the issue, changing this to `500` will affect all limit in the SDK

Since this affects *only* the `all().fetch()` method, you can add the limit params [here](https://github.com/sphereio/sphere-node-sdk/blob/5437331aa292449d6bbeaf7309d6c335e3b1fdf8/src/coffee/services/base.coffee#L321)